frontend/src/components/SubscribeMail/SubscribeMail.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import mailImg from '../../assets/images/default-mail.svg'
+import { FaExclamationTriangle } from 'react-icons/fa';
+import './SubscribeMail.styles.sass'
+
+export default function () {
+  const [name, setName] = useState('')
+  const [email, setEmail] = useState('')
+  const [isValid, setIsValid] = useState(true);
+  const [successMessage, setSuccessMessage] = useState('');
+  
+ 
+  const handleSubmit = e => {
+    e.preventDefault();
+    setSuccessMessage('')
+    setIsValid(true)
+    if (name.length <= 3 || email.length <= 3) {
+      setIsValid(false);
+    }else {
+      const subscriber = { name, email };
+      fetch('http://localhost:3000/api/subscription', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(subscriber)
+      })
+      .then(response => response.status === 201 ? setSuccessMessage('success') : setSuccessMessage('error'))
```

Comment:
@tamas-w the message could come from the backend actually and just display the message from the response